### ISSUE TITLE
US4C - implement contextual indoor directions when accessibility mode is on

### ISSUE THREAD

**Issue by gisellemartel** [gisellemartel/CONPASS#125]
Implement indoor directions based on start and end point set in the contextual indoor search bar. Note that directions should be based on accessibility needs (always selects elevator over escalator/stairs and accessible washroom)
